src/Lib.P2P.Tests/SwarmTest.cs

```diff
@@ -0,0 +1,1244 @@
+#region LICENSE
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [This file has 1087 lines, which is greater than 1000 authorized. Split it into smaller files.](https://app.codacy.com/manual/catalyst-network/Catalyst.Framework/pullRequest?prid=4832783)